pkg/envoy/sds/response.go

```diff
@@ -66,100 +59,30 @@ func (s *sdsImpl) getSDSSecrets(cert *certificate.Certificate, requestedCerts []
 			continue
 		}
 
-		log.Debug().Str("proxy", proxy.String()).Msgf("Proxy requested cert %s", requestedCertificate)
-
 		switch sdsCert.CertType {
```

Comment:
nit: a switch case is overkill here. just doing ` if sdsCert.CertType == secrets.RootCertTypeForMTLSOutbound {...}`